README.md

```diff
@@ -109,3 +71,45 @@ The following files will be created:
 - `index.html`: start page for the project
 - `pyscript.toml`: project metadata and config file
 - `main.py`: a "Hello world" python starter module
+
+#### Use --wrap to embed a python file OR a command string
+
+- ##### Embed a Python script into a PyScript HTML file
+
+```shell
+$ pyscript create --wrap <filename.py>
+```
+
+This will generate a project i.e. a new directory named `filename` under the current directory.
+
+Similar to the above, interactive prompts will further ask for metadata information.
+
+The following files will be created:
+- `filename.html`: embeds the code of `filename.py` in a `<py-script>` tag.
+- `pyscript.toml`: project metadata and config file
+
+This can be overridden with the `-o` or `--output` option:
+
+```shell
+$ pyscript create --wrap <filename.py> -o <another_filename.html>
+```
+
+i.e. the HTML file created in the above directory will now be named `another_filename.html`
+
+- ##### Very simple command examples with `--command` option
+
+The `-c` or `--command` option can be used to demo very simple cases.
+
+Note that `-o/--output` option is to be compulsorily used with the `-c/--command` option.
+
+```shell
+$ pyscript create --wrap -c 'print("Hello World!")' -o <output_filename.html>
+```
+
+This will generate a project i.e. a new directory named `output_filename` under the current directory.
+
+Similar to the above, interactive prompts will further ask for metadata information.
+
+The following files will be created:
```

Comment:
same comments as above....  it may be ok to create an inline option too where we don't create the python file and put the code within a `<script type="pyodide">` tag but that should be an option.

In general, we want to always try to encourage best practices

Reply:
Yeah, older `wrap` always made it inline.
And `create` used to create project folders.

So the current behaviour is the blend of both.

Asking again as asked in the first comment, so we don't want inline code ideally?
aka we want it to be via `<py-script src=...>` instead? Aka we do need the python file for `wrap`?

I am asking for a confirmation here since `wrap` was used to embed it as inline.

Reply:
Done!